infra/cluster-init/init.sh

```diff
@@ -47,14 +47,14 @@ function install_knative {
     kubectl apply \
         -f "https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION:-1.0.1}/serving-hpa.yaml"
 
-    kubectl apply \
-        -f knative-configmaps/config-features.yaml
-
     local core_apps=("activator" "autoscaler" "controller" "webhook")
     for app in ${core_apps[@]}; do
         kubectl wait -n knative-serving --for=condition=ready pod -l app=${app} --timeout=5m
     done
 
+    kubectl apply \
+        -f knative-configmaps/config-features.yaml
```

Comment:
Reordered this `cluster-init` step to ensure that the config map only gets applied when all the core components of knative have been up and running. This was causing flaky e2e cluster set up errors in the past.